playlist.c

```diff
@@ -2769,6 +2780,55 @@ void playlist_get_db_name(playlist_t *playlist, size_t idx,
    }
 }
 
+core_info_t *playlist_entry_get_core(const struct playlist_entry* entry)
+{
+   core_info_ctx_find_t core_info;
+
+   core_info.inf = NULL;
+   core_info.path = entry->core_path;
+
```

Comment:
Hmm... I would have thought that `if (path_is_valid(entry->core_path))` then we could skip the whole search operation... but I guess this is okay :)